marklogic-client-api/src/test/java/com/marklogic/client/test/ssl/OneWaySSLTest.java

```diff
@@ -85,10 +85,24 @@ void defaultSslContext() throws Exception {
 		assertTrue(ex.getCause() instanceof SSLException, "Unexpected cause: " + ex.getCause());
 	}
 
-	// Currently failing on 12-nightly due to https://progresssoftware.atlassian.net/browse/MLE-17505 .
 	@Test
-	@ExtendWith(RequiresML11OrLower.class)
-	void noSslContext() {
+	void noSslContext() throws Exception {
+		SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+		sslContext.init(null, new TrustManager[]{Common.TRUST_ALL_MANAGER}, null);
+		DatabaseClient sslClient = Common.newClientBuilder()
+			.withUsername(Common.SERVER_ADMIN_USER)
+			.withPassword(Common.SERVER_ADMIN_PASS)
+			.withSSLContext(sslContext)
```

Comment:
Try it with just `.withSSLProtocol("TLSv1.2")`, you shouldn't need to build an SSLContext yourself, the builder wll do it for you and will use the "trust all" trust manager.